src/Stepper/Step.less

```diff
@@ -0,0 +1,174 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+@stepIconWidth: 1.75rem; // 28px
+@iconBorderWidth: @size_4xs; // 1px
+@iconFocusShadowThickness: @size_2xs; // 4px
+@titleFontSize: 1.25rem; // 20px
+@alertSize: 0.375rem; // 6px
+
+.Step {
+  .flexbox();
+  .alignItems(flex-start);
+  width: 100%;
+  .padding--y--2xs;
+}
+
+.Step--button {
+  .button--reset();
+  .padding--none;
+  text-align: left;
+  .text--line-height-3;
+}
+
+.Step--title {
+  overflow: hidden;
+  color: @neutral_gray;
+  font-size: @titleFontSize;
+  .text--semi-bold;
+  .margin--top--2xs;
+  -ms-flex: 0 1 auto;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--description {
+  color: @neutral_gray;
+  .text--medium;
+  .text--regular;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--message--optional {
+  overflow: hidden;
```

Comment:
why does the message need an `overflow: hidden`? just making sure we cut down on the noise with unnecessary rules